### PR TITLE
EYQG 001 - 006

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
The first six mss from Enda Yohannes.

### CODE REVIEW

File: Enda Yohannes Qolla Gable/EYQG001.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1956" notAfter="1970"/>.     
+                             The manuscript was commisioned by wayzaro Zanabaš Walda Giyorgis and donated by her to
```
Comment (eu-genia) [inline Enda Yohannes Qolla Gable/EYQG001.xml:75]: commissioned (ss)

Comment (eu-genia) [reply Enda Yohannes Qolla Gable/EYQG001.xml:75]: Zanabaš or Zannabaš ?

File: Enda Yohannes Qolla Gable/EYQG002.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1950" notAfter="1999"/>.     
+                             The manuscript was an individual and donated by him to <placeName ref="INS1008EndaYohannes">ʾƎndā Yoḥannǝs Māy Kʷāḵʷ</placeName>.
```
Comment (eu-genia) [inline Enda Yohannes Qolla Gable/EYQG002.xml:74]: had been **owned by** an individual?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: Enda Yohannes Qolla Gable/EYQG001.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply Enda Yohannes Qolla Gable/EYQG001.xml:75]: ok

File: Enda Yohannes Qolla Gable/EYQG002.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply Enda Yohannes Qolla Gable/EYQG002.xml:74]: corrected

**Comment by eu-genia** [comment]
@line-o sorry can you check why the test has failed? I cannot see the error reported, and in Oxygen all is green. thanks!

File: Enda Yohannes Qolla Gable/EYQG001.xml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                                  <term key="MoPe"/> 
+                    <term key="Liturgy"/>
+                    <term key="Rituals"/>
```
Comment (thea-m) [inline Enda Yohannes Qolla Gable/EYQG001.xml:97]: Hagiography?

File: Enda Yohannes Qolla Gable/EYQG002.xml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                         </bindingDesc>
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1950" notAfter="1999"/>.     
+                             The manuscript was commissioned an individual and donated by him to <placeName ref="INS1008EndaYohannes">ʾƎndā Yoḥannǝs Māy Kʷāḵʷ</placeName>.
+                             The name other persons,  Walatta  ʿAbiya ʾƎgziʾ, Gabra Yoḥannǝs are mentioned in the supplication formulas.
```
Comment (thea-m) [inline Enda Yohannes Qolla Gable/EYQG002.xml:75]: names of

File: Enda Yohannes Qolla Gable/EYQG003.xml [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1700" notAfter="1799"/>.     
+                             The hand is similar to that of .
```
Comment (thea-m) [inline Enda Yohannes Qolla Gable/EYQG003.xml:73]: reference missing?

**Review by thea-m — CHANGES_REQUESTED** [review]
Nice :)

File: Enda Yohannes Qolla Gable/EYQG001.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply Enda Yohannes Qolla Gable/EYQG001.xml:97]: ok

File: Enda Yohannes Qolla Gable/EYQG002.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply Enda Yohannes Qolla Gable/EYQG002.xml:75]: ok

File: Enda Yohannes Qolla Gable/EYQG003.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply Enda Yohannes Qolla Gable/EYQG003.xml:73]: yes, sorry.

**Comment by eu-genia** [comment]
Now I know what is wrong. Your folder name has spaces in it.
You may never ever ever have file names or folder names with spaces.

**Pull request merged**